src/features/midi/utils.tsx

```diff
@@ -0,0 +1,62 @@
+import { Midi } from "@tonejs/midi"
+import fs from 'fs'
+import { Song, Track, Tracks, Bpm, SongNote } from '@/types'
+
+export function convertBase64MidiToSong(base64Midi: string) {
+    const binaryMidi = Buffer.from(decodeURIComponent(base64Midi), 'base64')
+    console.log(binaryMidi)
+    const midi = new Midi(binaryMidi)
```

Comment:
I just created a new `parseMidi` in https://github.com/sightread/sightread/pull/100 that you should be able to use here instead

Reply:
Done